back/engines/commercial/flag_inappropriate_content/app/views/flag_inappropriate_content/email_campaigns/inappropriate_content_flagged_mailer/campaign_mail.mjml

```diff
@@ -1,40 +1,39 @@
-<mj-section padding="25px"
-            text-align="left">
-  <mj-column
-    padding="20px 25px"
-    border="1px solid #eaeaea">
-    <mj-text>
-      <p style="font-size: 14px; margin-top: 10px; color: #84939E;">
-        <%= format_message('author_wrote', values: {authorName: event.flaggable_author_name }) %>
-      </p>
-      <% case event.flaggable_type %>
-      <% when 'Comment' %>
-        <h2 style="font-size: 18px; margin-top: 10px; margin-bottom: 10px;">
-          <%= localize_for_recipient(event.flaggable_body_multiloc) %>
-        </h2>
-      <% when 'Idea' %>
-        <h2 style="font-size: 18px; margin-top: 10px; margin-bottom: 10px;">
-          <%= localize_for_recipient(event.flaggable_title_multiloc) %>
-        </h2>
-        <p style="font-size: 14px; margin-top: 10px; color: #84939E;">
-          <%= localize_for_recipient(event.flaggable_body_multiloc) %>
-        </p>
+<mj-section padding="0 25px 25px">
```

Comment:
This fixes issues with double borders
<img width="621" alt="Screenshot 2025-05-21 at 19 10 03" src="https://github.com/user-attachments/assets/545231d8-5050-4c49-a383-5e68ace60198" />